muffnn/core.py

```diff
@@ -7,6 +7,52 @@
 from tempfile import NamedTemporaryFile
 import tensorflow as tf
 
+def affine_lasso(input_tensor, output_size, bias=True, bias_start=0.0,
```

Comment:
You don't need this function. You can just use the `affine` function with `bias=False`.